build.gradle

```diff
@@ -112,7 +112,10 @@ subprojects {
   }
 
   ext."signing.keyId" = "2EA0A67F"
-  ext."signing.password" = System.getenv("SIGNING_PASSWORD")
+  if (System.getenv("SIGNING_PASSWORD")) {
+    // if the password property is set, even if its null null, the SigningExtension will try to load the key
```

Comment:
`null null`?

What in the situation there is no password needed for a key?  `ext."signing.password"` should be set to null or has to be not set at all?